### ISSUE TITLE
ADSIEDIT/ ADUC / AD Admin Center ? 

### ISSUE THREAD

**Issue by PsychoData** [MicrosoftDocs/OfficeDocs-Exchange#2388]
Hi, 

I have a lot of people who see `whether a third-party management tool or ADSIEDIT can be` in the [third-party management tools](https://docs.microsoft.com/en-us/exchange/decommission-on-premises-exchange#can-third-party-management-tools-be-used) section, and think that as long as they are using AD Users and Computers, or AD Administrative Center, they are fine (since they are not using ADSIEDIT) - could this possibly be elaborated to include these two tools provided my Microsoft as unsupported? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a6eb1f9b-27f6-4d93-5a97-ad3f8b6ed27d
* Version Independent ID: f248ffa0-1bc3-be8f-055a-1c941e3ad234
* Content: [How and when to decommission your on-premises Exchange servers in a hybrid deployment](https://docs.microsoft.com/en-us/exchange/decommission-on-premises-exchange)
* Content Source: [Exchange/ExchangeHybrid/decommission-on-premises-exchange.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeHybrid/decommission-on-premises-exchange.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by scanum** [comment]
@PsychoData Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by PsychoData** [comment]
I don't have any support issues or problems with my tenants. 
I have colleagues at other companies getting confused because this talks about using ADSIEDIT to make changes to attributes on regular user objects and similar. 

This not-only is confusing to them because ADSIEDIT, ADUC, and ADAC are first-party tools from Microsoft, though not supported for Exchange changes, but also because most of them are not familiar with using ADSIEdit for "normal" attribute changes. 

This mention of ADSIEdit instead of a more common/"normal" admin tool for attribute changes - like ADUC - can cause people to end up trying to edit something in ADSIEdit that they shouldn't - like friends I have assisted who were trying to edit the Schema of the proxyAddresses field because they were trying to adjust the aliases on an Exchange account. 

For example, I had made changes to Aliases and some display name attributes in ADUC because I was already in there before - those changes were picked up by Exchange Hybrid and sent up, but AADConnect would also pick them up, and send them through to AzureAD fine, where they get picked up by Exchange Online and implemented. 
When I found this I was very confused because I only ever used ADSIEdit to make changes to directory partitions or deep domain configuration, so I wasn't even sure what sort of User-management someone might be doing with ADSIEdit Directly
